src/ucp/wireup/wireup_ep.c

```diff
@@ -409,17 +410,25 @@ ucp_rsc_index_t ucp_wireup_ep_get_aux_rsc_index(uct_ep_h uct_ep)
 
 ucs_status_t ucp_wireup_ep_connect(uct_ep_h uct_ep, const ucp_ep_params_t *params,
                                    ucp_rsc_index_t rsc_index, int connect_aux,
-                                   const ucp_unpacked_address_t *remote_address)
+                                   const ucp_unpacked_address_t *remote_address,
```

Comment:
ucp_wireup_ep_connect is intended to connect aux transport
need to use uct_ep_connect_to_ep() instead

Reply:
then why it has `connect_aux` flag?